Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -532,8 +532,11 @@ def audit_conflicts
       end
     end
 
+    versioned_conflicts_whitelist = %w[node@ bash-completion@]
+
     return unless formula.conflicts.any? && formula.versioned_formula?
-    return if formula.name.start_with? "node@"
+    versioned_conflicts_whitelist.each { |c|
+      return if formula.name.start_with? c }
```

Comment:
> I wasn't certain if homebrew used "splat" much

I think it is, but even if it wasn't, I would say in this case it is certainly more readable.